pipelines/nextflow/modules/genome_metadata/dump_ncbi_stats.nf

```diff
@@ -0,0 +1,68 @@
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+process DUMP_NCBI_STATS {
+    tag "${db.species}"
+    label "local"
+    maxForks 1
+    storeDir "$cache_dir/ncbi_stats/${db.species}"
+
+    input:
+        val server
+        val db
+        val cache_dir
+
+    output:
+        tuple val(db), path("ncbi_stats.json")
+
+    shell:
+        output = "ncbi_stats.json"
+        '''
+        function get_meta_value {
+            meta_key=$1
+
+            password_line=""
+            if [ "!{server.password}" != "" ]; then
+                password_line='--password !{server.password}'
+            fi
+            mysql --host '!{server.host}' \
+                --port '!{server.port}' \
+                --user '!{server.user}' \
+                $password_line \
+                --database '!{db.database}' \
```

Comment:
New trick I have just learned:
```suggestion
            mysql --host="!{server.host}" --port="!{server.port}" --user="!{server.user}" \
                --password="!{server.password}" --database="!{db.database}" \
```
This works even if the password is meant to be empty (like for `ensro`).